source/common/network/happy_eyeballs_connection_impl.cc

```diff
@@ -16,6 +16,7 @@ HappyEyeballsConnectionImpl::HappyEyeballsConnectionImpl(
           {source_address, socket_factory, transport_socket_options, options}),
       next_attempt_timer_(dispatcher_.createTimer([this]() -> void { tryAnotherConnection(); })) {
   ENVOY_LOG(trace, "New connection.");
```

Comment:
I wonder if we still need this log given the new logs you're adding?

Reply:
Didn't want to just delete them in case someone was paying attention to them, but since (I think) you put them here in the first place, then probably not. :)